src/open_inwoner/components/templates/components/File/File.html

```diff
@@ -1,13 +1,22 @@
-{% load i18n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
+{% load i18n l10n link_tags utils icon_tags form_tags dropdown_tags button_tags string_tags %}
 
 <aside class="file" aria-label="{% trans "Bestand" %} {{ name }}">
     <div class="file__file">
-    {% if is_image %}
-        {% icon icon="image" outlined=True %}
-    {% else %}
-        {% icon icon="description" outlined=True %}
-    {% endif %}
+        <p class="file__symbol">
+            {% if is_image %}
+                {% icon icon="image" outlined=True %}
+            {% else %}
+                {% icon icon="description" outlined=True %}
+            {% endif %}
+        </p>
+
+
     <p class="p">
+        {% if recently_added %}
+            {% icon icon="fiber_manual_record" outlined=False extra_classes="file__file--recent" %}
+            <span class="file__file--recent">{% trans "Nieuw" %}</span>
+            <br>
```

Comment:
Remove all BR's and give the surrounding paragraph a display Flex + flex-direction column so you can control the spacing.